pages/_app.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import NextApp from 'next/app'
+
+import Layout from '../components/layout'
+
+class App extends NextApp {
+  static async getInitialProps({ Component, ctx }) {
+    let pageProps = {}
+
+    if (Component.getInitialProps)
+      pageProps = await Component.getInitialProps(ctx)
+
+    return { pageProps }
+  }
+
+  render() {
+    const { Component, pageProps } = this.props
+
+    return (
+      <Layout>
+        {/* eslint-disable-next-line react/jsx-props-no-spreading */}
+        <Component {...pageProps} />
+      </Layout>
```

Comment:
I expected to bootstrap all application behaviour and layout here. Any ideas/objectives?

Reply:
I followed the same pattern as is in @oacore/next repository. I kind of like it as is but don't mind to move layout directly here.